### PR TITLE
Bug 1732193: UPSTREAM: 80518: Fix detachment of deleted volumes

### PR DESCRIPTION
**Pull request by jsafrane**
https://bugzilla.redhat.com/show_bug.cgi?id=1732193
upstream: https://github.com/kubernetes/kubernetes/pull/80518

cc @openshift/storage

### CODE REVIEW

File: vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/openstack/openstack.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ import (
 	"github.com/mitchellh/mapstructure"
 	"gopkg.in/gcfg.v1"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (bertinatto) [inline vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/openstack/openstack.go]: nit: only change in the file

Comment (jsafrane) [reply vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/openstack/openstack.go]: Interesting, it has been merged upstream this way. I removed it from this backport.

**Pull request merged**